### DISCUSSION TITLE
关于node版本管理的一个小建议

### DISCUSSION THREAD

**Discussion by SherkeyXD** [ArkAlliance/ArkAPI#1] [General]
使用fnm而不是nvm来进行node的版本管理

[fnm](https://github.com/Schniz/fnm)是一个rust编写的跨平台node版本管理工具，定位与nvm相同，但是效率更高

**Comment by GalvinGao** [comment]
已更换 README 中的相关提示，感谢建议！
（我自己也开始用了hhh
